NotificationService/AppGroupUserDefaultsStore.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  AppGroupUserDefaultsStore.swift
+//  RealifeTech-SDK
+//
+//  Created by YOU-HSUAN YU on 2022/6/27.
+//
+
+import Foundation
+
+enum AppGroupStoreKey: String {
+    case prefix = "NotificationExtension"
+    case configurationKey = "SDKConfiguration"
+}
+
+class AppGroupUserDefaultsStore {
+
+    private let appGroupStore: CodableStore
+    private let configurationKey = AppGroupStoreKey.configurationKey.rawValue
+
+    init?(appGroupId: String) {
```

Comment:
What exactly is the appGroupId?
Is there any way we can hide it from the integrator, so they don't have to know about it?